### PR TITLE
Clearing all translations enters reading mode

### PR DESCRIPTION
**Pull request by muhsinali**
Deselecting all translations enters reading mode, manually re-entering a translation again brings back the translation view.

### CODE REVIEW

File: src/scripts/components/header/ContentDropdown.js [hunk 1]
```diff
@@ -49,6 +49,20 @@ class ContentDropdown extends React.Component {
     this.context.executeAction(AyahsActions.updateAyahs, {
       content: chosenOptions
     });
+
+    if(chosenOptions.toString() === ''){
```
Comment (mmahalwy) [inline src/scripts/components/header/ContentDropdown.js]: Let's keep the coding style the same. Spaces between `if` and `(`, so `if (` and the curl `) {`.

Thanks!

Comment (mmahalwy) [inline src/scripts/components/header/ContentDropdown.js]: Also, chosenOptions should be an array no? So why not `.length === 0` ?

**Comment by mmahalwy** [comment]
Looks good mashallah :)

I know this may tire you a tad bit and I apologize but I do mean it for the best and for your learning inshallah, and that is, why not try to write a test for `ContentDropdown`? It's self contained enough and should be easy to unit test, but at the same time it will give you great experience to work with the unit tests + react which looks like you're getting a good handle of.

Check this out: 
https://github.com/quran/quran.com-frontend/pull/25/files

You will want to stub out (which means, not allow the function to propagate but spy on what it does and receives, comes from Sinon.js which is a great tool) the `executeAction` function and go from there. `ContentDropdown` should be VERY similar to `SearchInput`.

To run tests watching: `karma start` will do the trick. You don't need to run `npm run watch` or anything.

**Comment by muhsinali** [comment]
Not a problem at all! I turn in my project tomorrow for school insha'Allah and after that I'll be free

**Comment by mmahalwy** [comment]
@muhsinali sounds good! I am excited :)
We have introduced many bugs lately (I did with search) so i think it's time we write more tests so things don't break (I am really good at that, users are not happy with that haha).

**Comment by muhsinali** [comment]
Fixed the code! Tomorrow I'll get busy on the tests insha'Allah as this school project will finally be over with!

**Comment by mmahalwy** [comment]
+1

**Pull request merged**